src/test/java/com.vite/cli/ViteCLITest.java

```diff
@@ -0,0 +1,61 @@
+package com.vite.cli;
+
+import com.vite.service.Client;
+import com.vite.service.DefaultClient;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * Tests that the cli works as expected
+ */
+public class ViteCLITest extends org.hamcrest.Matchers {
+    private CLI cli;
+    private ByteArrayOutputStream outputStream;
+    private InputStream inputStream;
+    private static ClassLoader classLoader;
+    private static Client client;
+
+    @BeforeClass
+    public static void setClassLoader() {
+        classLoader = Thread.currentThread().getContextClassLoader();
+        client = new DefaultClient();
+    }
+
+    @Before
+    public void setup() {
+        inputStream = null;
+        outputStream = new ByteArrayOutputStream();
+        cli = new ViteCLI(System.in, outputStream, client);
+    }
+
+    /**
+     *Tests that help works as expected
+     */
+    @Test
+    public void testHelp() {
+        assertThat(cli.help("ping"), is(equalTo("ping\tPings the server\n")));
+        assertThat(cli.help("quit"), is(equalTo("quit\tExits the cli\n")));
+        assertThat(cli.help(), is(equalTo("ping\tPings the server\nquit\tExits the cli\n")));
+    }
+
+    /**
+     * Tests that ping works as expected
+     */
+    @Test
+    public void testPing() {
+        assertThat(cli.ping(), is(equalTo("")));
```

Comment:
This does not connect to the server. If you notice in setup I use the default instance of the client which doesn't connect to the server